flask_appbuilder/security/views.py

```diff
@@ -119,6 +121,15 @@ def form_post(self, form):
         flash(as_unicode(self.message), "info")
 
 
+def _roles_custom_formatter(string: str) -> str:
+    if current_app.config.get("AUTH_ROLES_SYNC_AT_LOGIN", False):
+        string += (
+            ". <b>Warning: AUTH_ROLES_SYNC_AT_LOGIN is enabled. "
+            "Changes to this field will not persiste."
```

Comment:
```suggestion
            "Changes to this field will not persist."
```
Just a typo! :) So happy to see this getting added!

Reply:
nice catch, thks!